index.js

```diff
@@ -18,19 +18,19 @@ const isPortFree = port => new Promise((resolve) => {
     server.close();
   });
 
-  server.listen(port, '0.0.0.0');
+  server.listen(port, host);
 });
 
-const findFreePort = port => isPortFree(port)
+const findFreePort = (port, host) => isPortFree(port, host)
   .then((isFree) => {
     if (isFree) {
       return port;
     }
 
-    return findFreePort(port + 1);
+    return findFreePort(port + 1, host);
   });
 
 module.exports = {
-  findFreePort: port => findFreePort(port),
-  isPortFree: port => isPortFree(port),
+  findFreePort: (port, host) => findFreePort(port, host),
```

Comment:
Can we use here short object notation or Node.js@12 hasn't supported it yet?

Reply:
Added short object notation.